<ui version="4.0" >
 <class>frmHeader</class>
 <widget class="QDialog" name="frmHeader" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>587</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>7</hsizetype>
    <vsizetype>7</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Header</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item rowspan="2" row="0" column="0" colspan="2" >
    <widget class="QSplitter" name="splitter1" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="splitter" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QGroupBox" name="groupBox1" >
       <property name="title" >
        <string>Comment</string>
       </property>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="0" column="0" >
         <widget class="QTextEdit" name="txtOtherComments" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>7</hsizetype>
            <vsizetype>7</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="tabChangesFocus" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox" >
      <property name="title" >
       <string>Header</string>
      </property>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item rowspan="5" row="0" column="0" >
        <widget class="QTableWidget" name="tableHeader" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>7</hsizetype>
           <vsizetype>7</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="mouseTracking" >
          <bool>false</bool>
         </property>
         <property name="focusPolicy" >
          <enum>Qt::TabFocus</enum>
         </property>
         <property name="contextMenuPolicy" >
          <enum>Qt::DefaultContextMenu</enum>
         </property>
         <property name="acceptDrops" >
          <bool>false</bool>
         </property>
         <property name="lineWidth" >
          <number>1</number>
         </property>
         <property name="midLineWidth" >
          <number>1</number>
         </property>
         <property name="tabKeyNavigation" >
          <bool>false</bool>
         </property>
         <property name="showDropIndicator" stdset="0" >
          <bool>false</bool>
         </property>
         <property name="dragEnabled" >
          <bool>false</bool>
         </property>
         <property name="alternatingRowColors" >
          <bool>true</bool>
         </property>
         <property name="selectionMode" >
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="showGrid" >
          <bool>true</bool>
         </property>
         <property name="columnCount" >
          <number>2</number>
         </property>
         <column/>
         <column/>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QPushButton" name="btnDown" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>1</hsizetype>
           <vsizetype>1</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip" >
          <string>Move down</string>
         </property>
         <property name="text" >
          <string>&amp;Down</string>
         </property>
         <property name="icon" >
          <iconset>../images/down.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QPushButton" name="btnUp" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>1</hsizetype>
           <vsizetype>1</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip" >
          <string>Move up</string>
         </property>
         <property name="statusTip" >
          <string/>
         </property>
         <property name="text" >
          <string>&amp;Up</string>
         </property>
         <property name="icon" >
          <iconset>../images/up.png</iconset>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QPushButton" name="btnInsertRow" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>1</hsizetype>
           <vsizetype>1</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font" >
          <font>
           <family>Sans Serif</family>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="toolTip" >
          <string>Insert row</string>
         </property>
         <property name="text" >
          <string> &amp;Insert</string>
         </property>
         <property name="icon" >
          <iconset>../images/add.png</iconset>
         </property>
        </widget>
       </item>
       <item row="4" column="1" >
        <widget class="QPushButton" name="btnDeleteRow" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>1</hsizetype>
           <vsizetype>1</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="palette" >
          <palette>
           <active>
            <colorrole role="WindowText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Button" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>221</red>
               <green>223</green>
               <blue>228</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Light" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Midlight" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Dark" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>85</red>
               <green>85</green>
               <blue>85</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Mid" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>199</red>
               <green>199</green>
               <blue>199</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Text" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="BrightText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="ButtonText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Base" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>239</red>
               <green>239</green>
               <blue>239</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Shadow" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Highlight" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>103</red>
               <green>141</green>
               <blue>178</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="HighlightedText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Link" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>0</red>
               <green>0</green>
               <blue>238</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="LinkVisited" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>82</red>
               <green>24</green>
               <blue>139</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="AlternateBase" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>232</red>
               <green>232</green>
               <blue>232</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="WindowText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Button" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>221</red>
               <green>223</green>
               <blue>228</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Light" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Midlight" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Dark" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>85</red>
               <green>85</green>
               <blue>85</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Mid" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>199</red>
               <green>199</green>
               <blue>199</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Text" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="BrightText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="ButtonText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Base" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>239</red>
               <green>239</green>
               <blue>239</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Shadow" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Highlight" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>103</red>
               <green>141</green>
               <blue>178</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="HighlightedText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Link" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>0</red>
               <green>0</green>
               <blue>238</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="LinkVisited" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>82</red>
               <green>24</green>
               <blue>139</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="AlternateBase" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>232</red>
               <green>232</green>
               <blue>232</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="WindowText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>128</red>
               <green>128</green>
               <blue>128</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Button" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>221</red>
               <green>223</green>
               <blue>228</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Light" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Midlight" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Dark" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>85</red>
               <green>85</green>
               <blue>85</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Mid" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>199</red>
               <green>199</green>
               <blue>199</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Text" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>199</red>
               <green>199</green>
               <blue>199</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="BrightText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="ButtonText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>128</red>
               <green>128</green>
               <blue>128</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Base" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>239</red>
               <green>239</green>
               <blue>239</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>239</red>
               <green>239</green>
               <blue>239</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Shadow" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Highlight" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>86</red>
               <green>117</green>
               <blue>148</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="HighlightedText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Link" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>0</red>
               <green>0</green>
               <blue>238</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="LinkVisited" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>82</red>
               <green>24</green>
               <blue>139</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="AlternateBase" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>232</red>
               <green>232</green>
               <blue>232</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="font" >
          <font>
           <family>Sans Serif</family>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="toolTip" >
          <string>delete row</string>
         </property>
         <property name="layoutDirection" >
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text" >
          <string> &amp;Remove</string>
         </property>
         <property name="icon" >
          <iconset>../images/minus.png</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>79</width>
           <height>51</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="resetButton" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="text" >
        <string>Re&amp;set</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton" >
       <property name="text" >
        <string>&amp;Apply Settings</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtOtherComments</tabstop>
  <tabstop>tableHeader</tabstop>
  <tabstop>btnUp</tabstop>
  <tabstop>btnDown</tabstop>
  <tabstop>btnInsertRow</tabstop>
  <tabstop>btnDeleteRow</tabstop>
  <tabstop>resetButton</tabstop>
  <tabstop>applyButton</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>frmHeader</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>frmHeader</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
